recipes/default.rb

```diff
@@ -46,6 +46,11 @@
   end
 end
 
+##  Set syncrepl_config dynamic values here
+node.default['openldap']['syncrepl_config']['searchbase'] = "\"#{node['openldap']['basedn']}\""
+node.default['openldap']['syncrepl_config']['binddn'] = "\"#{node['openldap']['syncrepl_cn']},#{node['openldap']['basedn']}\""
+node.default['openldap']['syncrepl_config']['credentials'] = "\"#{node['openldap']['slapd_replpw']}\""
```

Comment:
ah i think i see, you're relying on override attributes to override normal attributes...  yeah, please don't do that trick.